weave/integrations/google_ai_studio/google_ai_studio_sdk.py

```diff
@@ -5,12 +5,19 @@
 import weave
 from weave.trace.op_extensions.accumulator import add_accumulator
 from weave.trace.patcher import MultiPatcher, SymbolPatcher
+from weave.trace.serialize import dictify
 from weave.trace.weave_client import Call
 
 if TYPE_CHECKING:
     from google.generativeai.types.generation_types import GenerateContentResponse
 
 
+def gemini_postprocess_inputs(inputs: dict[str, Any]) -> dict[str, Any]:
```

Comment:
why is this needed? Shouldn't the to_json in call start handle this now?

Reply:
It's because of this:
> Given the purpose of a repr, it seemed to me that if a custom one was made we should by default use that instead of dictify'ing. However, in the Gemini integration, we explicitly want the inputs and output dictify-ed so that we don't truncate the strings and can build the chat view from structured data.

In the latest iteration we no longer attempt to dict'ify if there is a custom repr, which the Gemini objects have, so we have to explicitly do it as part of the integration.

Reply:
I see. Interestingly, this makes me realize that this PR effectively has 2 distinct user-facing changes:
1. Post-processing for Gemini
2. A more general purpose input/output routine.

Given that #2 runs the risk of bigger payloads / aggressive dumps, would it make sense to split this to land Gemini first, which we know is less likely to need revision?

Reply:
OK, I changed the general inputs/output case to `use_dictify=False` and updated the PR.